azure-mgmt-servicebus/tests/test_azure_mgmt_servicebus_queue.py

```diff
@@ -32,11 +32,11 @@ def test_sb_queue_curd(self, resource_group, location):
         # Create a Namespace
         namespace_name = "testingpythontestcasequeue"
 
-        namespaceparameter = SBNamespace(location, {'tag1': 'value1', 'tag2': 'value2'}, SBSku(SkuName.standard))
-        creatednamespace = self.servicebus_client.namespaces.create_or_update(resource_group_name, namespace_name, namespaceparameter,None,True).output
+        namespaceparameter = SBNamespace(location=location, tags={'tag1': 'value1', 'tag2': 'value2'}, sku=SBSku(name=SkuName.standard))
+        creatednamespace = self.servicebus_client.namespaces.create_or_update(resource_group_name, namespace_name, namespaceparameter)._polling_method._operation.resource
         self.assertEqual(creatednamespace.name, namespace_name)
 
-        while (self.servicebus_client.namespaces.get(resource_group_name, namespace_name).provisioning_state != 'Succeeded'):
+        while self.servicebus_client.namespaces.get(resource_group_name, namespace_name).provisioning_state != 'Succeeded':
```

Comment:
If namespace is a long running operation, calling `result()` will wait and you don't need that. There is automatic blocking on `result()`